source/includes/inventory/scan.md

```diff
@@ -0,0 +1,14 @@
+# Scan parcels
+
+Scan a parcel, when it is recognized you should hear a single beep (preview it [here](https://clyp.it/irpoec3u))
```

Comment:
you must explain what recognized means

Reply:
we don't have to share the sound file, if it is a good choice (UX), the user will know what it means.